.github/workflows/publish-to-npm.yml

```diff
@@ -13,7 +13,7 @@ jobs:
             - uses: actions/setup-node@v2
```

Comment:
Within the build workflow we updated checkout and setup actions versions. Do we want to make this workflow consistent with the other?

https://github.com/adobe/aio-cli-plugin-api-mesh/blob/39e1c6ccba6f52e49180d5f5c459645f862270fa/.github/workflows/build.yml#L18-L19

Reply:
If this is tested on the build workflow, I would say yes to maintaining consistency.

Reply:
@kmaschi updated those in the publish yaml and updated the JSTools/publish to the latest